app/controllers/api/v1/items_controller.rb

```diff
@@ -9,7 +9,10 @@ def show
   end
 
   def create 
-    id = Item.last.id ||= 0
+    # id = Item.last.id ||= 0
+
+    id = Item.last.id if Item.last 
+    id = 0 if Item.last.nil?
```

Comment:
Found this was necessary if it was the FIRST item/merchant. In which case Item.last.id = error bc Item.last = nil. I'd love a single liner for this